fastapi/__init__.py

```diff
@@ -1,6 +1,6 @@
 """FastAPI framework, high performance, easy to learn, fast to code, ready for production"""
 
-__version__ = "0.42.0"
+__version__ = "0.42.1"
```

Comment:
You should rebase against the most recent release; I think some of the minor formatting changes (like the extra commas) will go away. Also, don’t worry about bumping the version, @tiangolo will take care of that before the next release.